keyserver/server.go

```diff
@@ -609,6 +625,28 @@ func (ks *Keyserver) step(step *proto.KeyserverStep, rs *proto.ReplicaState, wb
 	return
 }
 
+func (ks *Keyserver) takeOutOfRotation() {
+	for {
+		ch := make(chan os.Signal, 1)
+		signal.Notify(ch, syscall.SIGUSR1)
```

Comment:
Instead of calling `signal.Notify` fresh again every loop iteration, why don't you just call it once and keep reading from the channel?